src/app/notes/notes.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="flex">
+  <app-note-list
+    class="list-component"
+    [noteList]="noteList"
+    [displayedColumns]="displayedColumns"
+    (addNote)="noteService.addNote()"
```

Comment:
Same as Day 7 PR, move these service calls to functions, change getNoteCategories to a property instead of a function, and we should be able to use currentNote instead of noteService.currentNote